maintainers/scripts/update.nix

```diff
@@ -17,9 +17,9 @@
 
 let
   pkgs = import ./../../default.nix (
-    if include-overlays == false then
+    if !include-overlays then
```

Comment:
This will break when `include-overlays` is a list of overlays.

Reply:
dropped that commit